lib/manager/reducers/projects.js

```diff
@@ -346,7 +346,10 @@ const projects = (state: ProjectsState = defaultState, action: Action): Projects
       })
     }
     case 'RECEIVE_DEPLOYMENTS': {
-      const {deployments, feedSourceId, projectId} = action.payload
+      const {deployments: deploymentsSummaries, feedSourceId, projectId} = action.payload
+      // Deployment summaries don't include a project id, so we need to add one
+      const deployments = deploymentsSummaries.map(d => ({...d, projectId}))
```

Comment:
The new `deploymentSummaries` comes with a new field `numberOfFeedVersions` that should now go under the "# of feeds" column in the `DeploymentListItem` component. As written in the PR, the "# of feeds" column in the list of deployments shows N/A until the full deployment object is fetched.

Reply:
Thanks for catching this one! Appreciate the heads up about `numberOfFeedVersions`. We now create a fake array.